### PR TITLE
[engraving] Reduced  score use. Step 23

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/engraving/layout/layoutoptions.h [hunk 1]
```diff
@@ -40,6 +40,7 @@ struct LayoutOptions
     LayoutMode mode = LayoutMode::PAGE;
 
     bool isShowVBox = true;
+    double noteHeadWidth = 0.0;
```
Comment (cbjeukendrup) [inline src/engraving/layout/layoutoptions.h:43]: I don't think this should be in options. It is rather something similar to style. In fact, this value is fully derived from style settings, see `Score::spatiumChanged`. The only reason that it is a separate field, is for performance (caching). So maybe `LayoutConfiguration` itself is the best place for it.

File: src/engraving/layout/v0/systemlayout.h [hunk 2]
```diff
@@ -51,14 +51,16 @@ class SystemLayout
     static void restoreLayout2(System* system, LayoutContext& ctx);
     static void setMeasureHeight(System* system, double height, LayoutContext& ctx);
     static void layoutBracketsVertical(System* system, LayoutContext& ctx);
-    static void layoutInstrumentNames(System* system);
+    static void layoutInstrumentNames(System* system, LayoutContext& ctx);
 
     static void setInstrumentNames(System* system, LayoutContext& ctx, bool longName, Fraction tick = { 0, 1 });
 
+    static double minDistance(LayoutContext& ctx, const System* top, const System* bottom);
```
Comment (RomanPudashkin) [inline src/engraving/layout/v0/systemlayout.h:58]: LayoutContext& ctx should probably be the last argument here (for consistency)

**Review by RomanPudashkin — APPROVED** [review]

File: src/engraving/layout/v0/systemlayout.h (hunk 2 above)
Comment (igorkorsukov) [reply src/engraving/layout/v0/systemlayout.h:58]: In a next PR I will fix it

File: src/engraving/layout/layoutoptions.h (hunk 1 above)
Comment (igorkorsukov) [reply src/engraving/layout/layoutoptions.h:43]: In a next PR I will see what can be done about it.

**Pull request merged**